src/chat_do.py

```
@@ -0,0 +1,406 @@
+            return
+
+        # Lookup session
+        sid = None
+        info = None
+        for s_id, s_info in self.sessions.items():
+            try:
+                if s_info["ws"] is ws or s_info["ws"] == ws:
+                    sid, info = s_id, s_info
+                    break
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)